src/components/OptimismDowntimeWarning/index.tsx

```diff
@@ -44,18 +45,15 @@ export default function OptimismDowntimeWarning() {
     <Root>
       <TitleRow>
         <WarningIcon />
-        <Trans>{'Optimism'} Scheduled Downtimes</Trans>
+        <Trans>{'Optimism'} Scheduled Downtime</Trans>
```

Comment:
FYI `{'Optimism'}` didn't actually extract into a variable in the extraction step. :/